conf/config-default.yaml

```diff
@@ -63,7 +63,27 @@ apisix:
       - name: memory_cache
         memory_size: 50m
 
-  delete_uri_tail_slash: false        # Delete the '/' at the end of the URI
+  delete_uri_tail_slash: false    # delete the '/' at the end of the URI
+
+  # http is the default proxy mode. proxy_mode can be one of `http`, `stream`, or `http&stream`
+  proxy_mode: http
+  #stream_proxy:                  # TCP/UDP proxy
+  #  tcp:                         # TCP proxy port list
+  #    - addr: 9100
+  #      tls: true
+  #    - addr: "127.0.0.1:9101"
+  #  udp:                         # UDP proxy port list
+  #    - 9200
+  #    - "127.0.0.1:9201"
+  #dns_resolver:                  # If not set, read from `/etc/resolv.conf`
+  #  - 1.1.1.1
+  #  - 8.8.8.8
+  #dns_resolver_valid: 30         # if given, override the TTL of the valid records. The unit is second.
```

Comment:
The configuration is duplicated

Reply:
@monkeyDluffy6017 Where is it duplicated? Or which line is duplicated. I only see it once.

Reply:
<img width="656" alt="image" src="https://github.com/apache/apisix/assets/2706161/9a900339-cd17-42a0-9edc-8eed421886f8">

Reply:
I think you should keep the original one, it's been modified by @kayx23 recently